engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -67,6 +68,23 @@ public void setReason(String reason)
     this.reason = reason;
   }
 
+  public void setLogLevel(LogLevel logLevel)
+  {
+    this.logLevel = logLevel;
+  }
+
+  public LogLevel getLogLevel()
+  {
+    return logLevel;
+  }
+
+  public static enum LogLevel
+  {
+    INFO,
```

Comment:
They may be useful in the future. Consider adding TRACE, DEBUG and possibly FATAL.